src/layouts/MainLayout.vue

```diff
@@ -1,21 +1,116 @@
-<template>
-  <q-layout view="hHh lpR fFf">
-    <q-page-container>
-      <router-view />
-    </q-page-container>
-  </q-layout>
-</template>
-
 <script lang="ts">
-import { defineComponent } from 'vue';
+// import libraries
+import { setCssVar } from 'quasar';
+import { defineComponent, ref } from 'vue';
+import { i18n } from 'src/boot/i18n';
+
+// import components
+import VueDrawerHeader from 'components/VueDrawerHeader.vue';
+import VueUserSelect from 'components/VueUserSelect.vue';
+import VueDrawerMenu from 'components/VueDrawerMenu.vue';
+
+// import types
+import { User } from 'components/types';
+
+if (window.Cypress) {
+  window.i18n = i18n;
+}
+
+const rideToWorkByBikeConfig: object = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG
+);
+setCssVar('gray-light', rideToWorkByBikeConfig.colorGrayLight);
 
 export default defineComponent({
   name: 'MainLayout',
 
-  components: {},
+  components: { VueDrawerHeader, VueUserSelect, VueDrawerMenu },
 
   setup() {
-    return {};
+    const leftDrawerOpen = ref(false);
+
+    const users: User[] = [
+      {
+        label: 'User 1',
+        value: '1',
+        image: 'https://picsum.photos/id/40/300/300',
+      },
+      {
+        label: 'User 2',
+        value: '2',
+        image: 'https://picsum.photos/id/64/300/300',
+      },
+      {
+        label: 'User 3',
+        value: '3',
+        image: 'https://picsum.photos/id/91/300/300',
+      },
+    ];
+
+    const toggleLeftDrawer = () => {
+      leftDrawerOpen.value = !leftDrawerOpen.value;
+    };
+
+    return {
+      leftDrawerOpen,
+      users,
+      toggleLeftDrawer,
+    };
   },
 });
 </script>
+
+<template>
```

Comment:
Add simple comments for better readability/orientation inside template code, please.

Reply:
Added comments